src/views/ExploreObservatories.vue

```diff
@@ -1,9 +1,191 @@
+          <div class="item-inner">
+            <img
+              src="https://images.unsplash.com/photo-1508615039623-a25605d2b022?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2250&q=80"
+              alt="#"
+            />
+          </div>
+        </li>
+        <li class="menu-item">
+          <span>
+            <GradientText theme="yellow-to-gold">IV</GradientText>
+          </span>
+          <div class="item-inner">
+            <img
+              src="https://images.unsplash.com/photo-1508615039623-a25605d2b022?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2250&q=80"
+              alt="#"
+            />
+          </div>
+        </li>
+        <li class="menu-item">
+          <span>
+            <GradientText theme="yellow-to-gold">V</GradientText>
+          </span>
+          <div class="item-inner">
+            <img
+              src="https://images.unsplash.com/photo-1508615039623-a25605d2b022?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2250&q=80"
+              alt="#"
+            />
+          </div>
+        </li>
+        <li class="menu-item">
+          <span>
+            <GradientText theme="yellow-to-gold">VI</GradientText>
+          </span>
+          <div class="item-inner">
+            <img
+              src="https://images.unsplash.com/photo-1508615039623-a25605d2b022?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2250&q=80"
+              alt="#"
+            />
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
 </template>
 
 <script>
-export default {};
+import GradientText from "@/components/GradientText";
+
+export default {
+  name: "ExploreObservatories",
+  components: {
+    GradientText,
+  },
+};
 </script>
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+@import "@/theme/colors.scss";
+@import "@/theme/media.scss";
+@import "@/theme/sizing.scss";
+@import "@/theme/typography.scss";
+
+header {
+  display: flex;
+  justify-content: center;
+  width: 100vw;
+  height: rem(150px);
+  margin-bottom: rem(60px);
+
+  img {
+    width: 100%;
+  }
+}
+
+.explore-observatories-copy {
+  @include robotoLight;
+
+  display: flex;
+  justify-content: center;
+}
+
+.copy-text {
+  width: 90vw;
+
+  p {
+    color: $silver;
+    font-size: rem(16px);
+    text-transform: uppercase;
+    letter-spacing: rem(8px);
+    text-align: center;
+    line-height: 1.8;
+    margin-bottom: rem(50px);
+  }
+
+  @include media(">=tablet") {
+    width: 70vw;
+    font-size: rem(28px);
+  }
+}
+
+.menu-wrapper {
+  display: flex;
+  width: 100vw;
+  justify-content: center;
+  margin: rem(30px) auto;
+}
+
+.menu-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: rem(50px);
+  list-style-type: none;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
```

Comment:
Just use `margin: 0` and `padding: 0`. These properties dont do what you think they do, block is attached to writing mode

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block-start